autoload/go/config.vim

```diff
@@ -211,6 +211,10 @@ function! go#config#DebugWindows() abort
 
 endfunction
 
+function! go#config#DebugPreserveLayout() abort
+  return get(g:, 'go_debug_preserve_layout', -1)
```

Comment:
Please make the default value `0`, as the documentation you added indicates it should be.

Reply:
see comment below